client/flutter/lib/components/home_page_sections/home_page_header.dart

```diff
@@ -1,172 +1,195 @@
                       ),
                     ),
                   ],
                 ),
-              ),
-              SizedBox(
-                height: 32,
-              ),
-              ThemedText(
-                this.title,
-                variant: TypographyVariant.title,
-                style: TextStyle(
-                  color: CupertinoColors.white,
-                ),
-                textAlign: TextAlign.left,
-              ),
-              ThemedText(
-                this.subtitle,
-                variant: TypographyVariant.body,
-                style: TextStyle(
-                  color: CupertinoColors.white,
-                ),
-              ),
-              Container(
-                height: 24,
-              ),
-              Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: [
-                  this.svgAssetName != null
-                      ? SvgPicture.asset(this.svgAssetName)
-                      : Container(),
-                  Padding(
-                    padding: EdgeInsets.symmetric(
-                      horizontal: 12,
-                    ),
-                    child: CupertinoButton(
-                      borderRadius: BorderRadius.circular(50),
-                      padding: EdgeInsets.symmetric(
-                        horizontal: 32,
-                        vertical: 8,
-                      ),
-                      color: CupertinoColors.white,
-                      child: ThemedText(
-                        this.buttonText,
-                        variant: TypographyVariant.button,
-                        style: TextStyle(color: Constants.primaryColor),
-                      ),
-                      onPressed: () {
-                        return this.link.open(context);
-                      },
-                    ),
-                  ),
-                ],
-              ),
-              if (this.svgAssetName == null)
-                Container(
-                  height: 40,
-                ),
+              )
             ],
           ),
-        ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildLogoSection() {
+    return Container(
+      constraints: BoxConstraints(minHeight: 48.0),
+      padding: EdgeInsets.symmetric(
+        horizontal: 24.0,
+      ),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          Container(
+            color: CupertinoColors.white,
+            height: 48.0,
+            width: 48.0,
+            margin: EdgeInsets.only(right: 8),
+            // TODO: Add WHO logo
+            // child: ,
+          ),
+          Container(
+            margin: EdgeInsets.symmetric(
+              horizontal: 14.0,
+            ),
+            color: Color.fromARGB(127, 250, 232, 169),
+            height: 48.0,
+            width: 1.0,
+          ),
+          Text(
```

Comment:
Should this be ThemedText

Reply:
Glad you pointed this out, because the style actually needs to change on this... the style of this text doesn't really align with any of our variants, so we'd be overwriting most of the style anyway. I think it's ok to leave this un-themed